docs/Seeding.md

```diff
@@ -1,7 +1,6 @@
 # Seeding the Database
 
-This covers seeding the database for development work.
-
+This covers seeding the database of public bodies for development work. "Public bodies" or "Public agencies" is the monicker for government agencies that are subject to the Uniform Information Practices Act (UIPA) in Hawaii. These include organizations such as the Department of Land & Natural Resources, Department of Planning & Permitting, and the Office of Information Practices.
```

Comment:
It might be clearer to say:
```
"Public bodies" or "Public agencies" means government agencies
```
 instead of:
```
"Public bodies" or "Public agencies" is the monicker for government agencies
```